orttraining/orttraining/python/training/ortmodule/_custom_autograd_function_exporter.py

```diff
@@ -204,68 +232,27 @@ def _export_pt_1_10(g, n, *args, **kwargs):
 def _post_process_after_export(exported_model, enable_custom_autograd_function):
     if enable_custom_autograd_function:
         return _post_process_enabling_autograd_fallback(exported_model)
-
-    is_pythonop_needed = False
-    for node in exported_model.graph.node:
-        if node.domain == "com.microsoft" and node.op_type in ["PythonOp"]:
-            is_pythonop_needed = True
-            break
-
-    if is_pythonop_needed:
-        warnings.warn(
-            "Detected autograd functions usage in current model, the run will fail \
-                      without enabling '_enable_custom_autograd_function'. Please enable it with: \
-                      'module._execution_manager(is_training_mode)._enable_custom_autograd_function = True'",
-            UserWarning,
-        )
-
```

Comment:
seems unrelated to the PR goal. Is that intentional?

Reply:
Oh I fortgot to mention, this is kind of dead code, that won't be trigger. If enable_custom_autograd_function, then this won't be run; otherwise, export will fail to handle autograd.Function because we did not register custom export function; So anyway, this won't be triggered. :)